Patches/MechanoidsExtraordinaire/ThingDefs_Races/Races_Mechanoid_MechanoidsExtraordinaire.xml

```diff
@@ -90,13 +91,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>Defs/ThingDef[defName="Mechanoid_Hound"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>12</ArmorRating_Blunt>
+					<ArmorRating_Blunt>35</ArmorRating_Blunt>
 				</value>
 			</li>
 			<li Class="PatchOperationReplace">
 				<xpath>Defs/ThingDef[defName="Mechanoid_Hound"]/statBases/ArmorRating_Sharp</xpath>
 				<value>
-					<ArmorRating_Sharp>8</ArmorRating_Sharp>
+					<ArmorRating_Sharp>15</ArmorRating_Sharp>
```

Comment:
This is already fast, pretty well-armed mech. Their melee attacks are only slightly weaker than a Scyther's and they have a grenade launcher. 8 sharp was sufficient to make them proof against anything smaller than 7.62mm NATO FMJ. At 15, they're proof against 7.62mm NATO AP as well.

If we go with this much of a change, we're going from having the preferred weapon against them going from a battle rifle with AP rounds to a heavy machine gun or anti-material rifle.

If you feel it needs to be increased, I wouldn't raise it much higher than 10 sharp.